meson.build

```diff
@@ -1466,12 +1451,43 @@ if want_dbus != 'false' and not skip_deps
         libdbus = dependency('dbus-1',
                              version : '>= 1.3.2',
                              required : want_dbus == 'true')
-        have = libdbus.found()
 else
-        have = false
-        libdbus = []
+        libdbus = disabler()
+endif
+conf.set10('HAVE_DBUS', libdbus.found())
+
+dbuspolicydir = get_option('dbuspolicydir')
+if dbuspolicydir == ''
+        if libdbus.found()
+                dbuspolicydir = libdbus.get_variable(pkgconfig: 'datadir', default_value: datadir) / 'dbus-1/system.d'
+        else
+                dbuspolicydir = datadir / 'dbus-1/system.d'
+        endif
+endif
+
+dbussessionservicedir = get_option('dbussessionservicedir')
+if dbussessionservicedir == ''
+        dbussessionservicedir = datadir / 'dbus-1/services'
+        if libdbus.found()
+                dbussessionservicedir = libdbus.get_variable(pkgconfig: 'session_bus_services_dir', default_value: dbussessionservicedir)
+        endif
+endif
+
+dbussystemservicedir = get_option('dbussystemservicedir')
+if dbussystemservicedir == ''
+        dbussystemservicedir = datadir / 'dbus-1/system-services'
+        if libdbus.found()
+                dbussystemservicedir = libdbus.get_variable(pkgconfig: 'system_bus_services_dir=', default_value: dbussystemservicedir)
```

Comment:
I think should just be `system_bus_services_dir` not `system_bus_services_dir=`.

Reply:
Ouch!